DirectEmployers/MyJobs
# [PD 2977] remove access

## What's this PR do?
This PR removes all references to Classic Reporting in Reports and strictly goes to dynamic reporting 100% of the time.

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start at the reports home page found in the navigation bar of secure.my.jobs

## How should this be manually tested?
Pull Down the Repository and go to the navigation bar clicking on Reports. Reports should not ask for or allow for you to go to Classic Reporting and should go straight to dynamic reporting view.

## Any background context you want to provide?
All Laughs from the Python guys are welcomed as far as the views.py file that was edited by the FRONT END Developer lol.
